static/js/ui.js

```diff
@@ -39,8 +39,12 @@ exports.reset_scrollbar = function (element_selector) {
     }
 };
 
-exports.destroy_scrollbar = function (element) {
-    element[0].perfectScrollbar.destroy();
+exports.destroy_scrollbar = function (element_selector) {
+    var element = element_selector[0];
+    if (element.perfectScrollbar !== undefined) {
```

Comment:
We expect this will always be true, right? If this were `master` I’d prefer to remove the conditional so we wouldn’t grow callers that rely on it, although for 2.0.x it doesn’t matter so much.

Reply:
I don't have the callstacks to show it, but i think i was hitting situations where rebuild_recent() was getting called triggering remove_expanded_private_messages where a scrollbar wasn't attached since that only happens at the end of update_private_messages